From 132989a22a43add52ccabdc908307e21bcdc819b Mon Sep 17 00:00:00 2001
From: Brandon Surowiec <BrandonSurowiec@users.noreply.github.com>
Date: Tue, 9 Mar 2021 09:07:07 -0500
Subject: [PATCH] Add missing 'after_commit' attribute

There's a new `after_commit` config option that isn't in the default config:
https://laravel.com/docs/8.x/queues#jobs-and-database-transactions

For new projects it may be better to have these defaulted to `true`, but I left it as `false` for now.
---
 config/queue.php | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/config/queue.php b/config/queue.php
index 4cef37cc14..25ea5a8193 100644
--- a/config/queue.php
+++ b/config/queue.php
@@ -39,6 +39,7 @@
             'table' => 'jobs',
             'queue' => 'default',
             'retry_after' => 90,
+            'after_commit' => false,
         ],
 
         'beanstalkd' => [
@@ -47,6 +48,7 @@
             'queue' => 'default',
             'retry_after' => 90,
             'block_for' => 0,
+            'after_commit' => false,
         ],
 
         'sqs' => [
@@ -57,6 +59,7 @@
             'queue' => env('SQS_QUEUE', 'default'),
             'suffix' => env('SQS_SUFFIX'),
             'region' => env('AWS_DEFAULT_REGION', 'us-east-1'),
+            'after_commit' => false,
         ],
 
         'redis' => [
@@ -65,6 +68,7 @@
             'queue' => env('REDIS_QUEUE', 'default'),
             'retry_after' => 90,
             'block_for' => null,
+            'after_commit' => false,
         ],
 
     ],
